specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-03-01-preview/subscription.json

```diff
@@ -0,0 +1,312 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-03-01-preview",
+    "title": "SubscriptionManagementClient",
+    "description": "Subscription management client provides an interface to create and manage Azure subscriptions programmatically."
```

Comment:
Merge with the description of the stable version. Drop the term "management client".